citrination_client/search/client.py

```diff
@@ -136,13 +136,13 @@ def pif_multi_search(self, multi_query):
         :return: :class:`PifMultiSearchResult` object with the results of the query.
         """
         failure_message = "Error while making PIF multi search request"
-        response_dict = self._get_success_json(self._post(routes.pif_multi_search, data=json.dumps(multi_query, cls=QueryEncoder), failure_message=failure_message))
+        response_dict = self._get_success_json(self._post(routes.pif_multi_search, data = json.dumps(multi_query, cls = QueryEncoder), failure_message = failure_message))
 
         return PifMultiSearchResult(**keys_to_snake_case(response_dict['results']))
 
-    def generate_simple_chemical_query(self, name=None, chemical_formula=None, property_name=None, property_value=None,
-                               property_min=None, property_max=None, property_units=None, reference_doi=None,
-                               include_datasets=[], exclude_datasets=[], from_index=None, size=None):
+    def generate_simple_chemical_query(self, name = None, chemical_formula = None, property_name = None, property_value = None,
```

Comment:
I believe PEP8 also has strong feelings about multiline function signatures. This one might be worth digging into.